### DISCUSSION TITLE
Carousel panels load slowly when using "Random" after latest release

### DISCUSSION THREAD

**Discussion by GopoBoi** [ppy/osu#26523] [Q&A]
https://github.com/ppy/osu/assets/69014805/540ca0b5-7b93-4bb2-a444-38886a8d9d40

Previous release for comparison:

https://github.com/ppy/osu/assets/69014805/bfee549f-3dbe-4fa6-b642-81068872672e

It's not a huge difference, but it's slow enough to be annoying.

**Comment by peppy** [comment]
Even in your videos, I think it seems fine?
